src/types.ts

```diff
@@ -16,11 +20,13 @@ export interface Scaled {
 
   width: number;
   height: number;
+
+  pageNumber: number;
 }
 
 export interface Position {
-  boundingRect: LTWH;
-  rects: Array<LTWH>;
+  boundingRect: LTWHP;
+  rects: Array<LTWHP>;
   pageNumber: number;
```

Comment:
Do we still need `pageNumber` here if all the rects have `pageNumber`?

Reply:
I believe you are correct.  I will check that out.

Reply:
I looked at this further and my thought is that `pageNumber` should probably stay for the purpose of backwards compatibility.  If it is removed, older versions of the highlights would not display correctly.  We store the highlights in a database and display them when a PDF is loaded.  I suspect a lot of other people do the same.